trace/collector.go

```diff
@@ -203,6 +199,11 @@ func (pc *PacketCountCollector) Process(t akinet.ParsedNetworkTraffic) error {
 			Unparsed:  1,
 		})
 	}
+	if pc.PacketCounts.Get().HTTPRequests > 0 && pc.PacketCounts.Get().HTTPResponses > 0 {
```

Comment:
It would be slightly better to do one Get().

I guess what I'm worried about is taking another trip through the mutex at all -- does that add a lot of overhead?  Is the copy that we make too much?  Should we instead add a new method that doesn't have to make a copy?

But even if all that is fine, it's twice as expensive to do it twice.  :)

Reply:
The alternative is a method something like `pc.PacketCounts().HasRequestAndResponse()` which can access the counts directly with the mutex held, so it's a single mutex call and doesn't require a copy of the PacketCount structure.  But that structure is relatively small?